src/main/java/io/stargate/sgv2/jsonapi/service/operation/tables/DropIndexAttempt.java

```diff
@@ -0,0 +1,41 @@
+package io.stargate.sgv2.jsonapi.service.operation.tables;
+
+import static io.stargate.sgv2.jsonapi.util.CqlIdentifierUtil.cqlIdentifierFromUserInput;
+
+import com.datastax.oss.driver.api.core.CqlIdentifier;
+import com.datastax.oss.driver.api.core.cql.SimpleStatement;
+import com.datastax.oss.driver.api.querybuilder.SchemaBuilder;
+import com.datastax.oss.driver.api.querybuilder.schema.Drop;
+import io.stargate.sgv2.jsonapi.service.cqldriver.executor.KeyspaceSchemaObject;
+import io.stargate.sgv2.jsonapi.service.operation.SchemaAttempt;
+import java.time.Duration;
+
+/*
+ An attempt to drop index for a column.
+*/
+public class DropIndexAttempt extends SchemaAttempt<KeyspaceSchemaObject> {
+  private final CqlIdentifier name;
+  private final boolean ifExists;
+
+  public DropIndexAttempt(
+      int position, KeyspaceSchemaObject schemaObject, CqlIdentifier name, boolean ifExists) {
+    super(position, schemaObject, new SchemaRetryPolicy(2, Duration.ofMillis(10)));
```

Comment:
Should we collect all these timeouts to one place, and/or possible have a single "default retry timeout duration" value to refer to (wrt last argument).
And possibly something like "default DDL retry max count"?

Reply:
Moving this to config based to attempt class. Missed it.